src/main/resources/db/migration/V1_0_0__Create_Location_Entity.sql

```diff
@@ -4,8 +4,8 @@ CREATE TABLE location
     country_code       VARCHAR(2)   NOT NULL,
     address            VARCHAR(255) NOT NULL,
     postal_code        VARCHAR(10)  NOT NULL,
-    latitude           DOUBLE,
-    longitude          DOUBLE,
+    latitude           MEDIUMINT,
+    longitude          MEDIUMINT,
```

Comment:
It would be better to do this change in a new file with higher version to prevent issues when deploying to AWS.